Riot/Modules/Room/RoomInfo/RoomInfoList/Cells/RoomInfoBasicTableViewCell.swift

```diff
@@ -0,0 +1,62 @@
+// 
+// Copyright 2020 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import Reusable
+import MatrixKit
+
+class RoomInfoBasicTableViewCell: GroupedTableViewCell {
+
+    @IBOutlet private weak var avatarImageView: MXKImageView!
+    @IBOutlet private weak var shadowView: UIView! {
+        didSet {
+            let shadowPath = UIBezierPath(roundedRect: shadowView.bounds, cornerRadius: 0)
```

Comment:
You have to move some of this code to `layoutSubviews` if this shadow needs to be redraw on orientation change for ex.

Reply:
Ah no, shadowView's size is constant.